### ISSUE TITLE
String keys do not work with given? and nested hash

### ISSUE THREAD

**Issue by ncullen** [orgsync/active_interaction#425]
```ruby
class Example < ActiveInteraction::Base
  hash :contact_info,  default: nil do
    string  :phone,    default: nil, strip: true
  end
  def execute
    given?(:contact_info, :phone)
  end
end
```

```
# Returns true (correct)
Example.run!(contact_info: { phone: '513-888-9999' })
# Returns false (incorrect)
Example.run!({"contact_info" => { "phone"=>"513-888-9999" } })
```

**Comment by ncullen** [comment]
I am using v3.5.2

**Comment by AaronLasseigne** [comment]
Fixed and released with v3.5.3. Thanks for letting me know.
